src/coreclr/jit/importer.cpp

```diff
@@ -4093,6 +4007,41 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
                 break;
             }
 
+            case NI_System_Activator_AllocatorOf:
+            case NI_System_Activator_DefaultConstructorOf:
+            case NI_System_Object_MethodTableOf:
+            case NI_System_EETypePtr_EETypePtrOf:
+            {
+                noway_assert(IsTargetAbi(CORINFO_CORERT_ABI)); // Only CoreRT supports it.
```

Comment:
```suggestion
                assert(IsTargetAbi(CORINFO_CORERT_ABI)); // Only CoreRT supports it.
```
Nit: I think this can be a regular assert.